### PR TITLE
Fixed newsfader options initialization

### PR DESCRIPTION
**Pull request by emanuele45**

### CODE REVIEW

File: themes/default/index.template.php [hunk 1]
```diff
@@ -811,5 +811,5 @@ function template_news_fader()
 		</ul>';
 
 	addInlineJavascript('
-		$(\'#elkFadeScroller\').Elk_NewsFader();', true);
+		$(\'#elkFadeScroller\').Elk_NewsFader(' . (empy($settings['newsfader_time']) : '' : '{\'iFadeDelay\': ' . $settings['newsfader_time'] . '}') . ');', true);
```
Comment (Spuds) [inline themes/default/index.template.php]: empTy would be better 😇

Comment (emanuele45) [inline themes/default/index.template.php]: darn...

Comment (Spuds) [inline themes/default/index.template.php]: And its missing a ? in that ternary ... put down the 🍷

**Comment by Spuds** [comment]
Still missing the ? in the  ternary ... More Grappa !

**Comment by emanuele45** [comment]
and two...

**Comment by emanuele45** [comment]
Looks like I didn't pick a good day for squashing some bugs... xD

**Pull request merged**